extensions/vi-mode/ex-command.lisp

```diff
@@ -190,6 +190,26 @@
             (lem:show-message (format nil "~A: ~S" option-name (encode-value option-value))
                               :timeout 10))))))
 
+(define-ex-command "^r(?:e|ead)?$" (range filename)
+  ;; TODO: range currently does not distinguish between :0 and :1.
+  ;; This makes it impossible to insert at the beginning of the file
+  (cond
+    ((string= "" filename)
+     (lem:message "No file name"))
```

Comment:
How about using lem:editor-error?
Errors in command execution are handled in the command loop and messages are displayed when using lem:editor-error.

Reply:
Thanks, fixed.